dataprofiler/tests/profilers/test_numeric_stats_mixin_profile.py

```diff
@@ -176,6 +176,8 @@ def test_timeit_merge(self):
         other2.min, other2.max, other2.variance, other2.sum = 1, 1, 1, 1
 
         # set auto as only histogram to merge
+        other1.histogram_selection = "auto"
```

Comment:
this error'ed without it?

Reply:
Yes because histogram selection is used whether to determine if the histogram exists or not (deciding, is it mergeable or not)

Reply:
Can you test setting it to None, which is the base? Or do we have a test which evaluates merging two profiles which are trained on empty data? Just want to make sure `None` (default value) doesn't cause a failure.

Reply:
This test may exist.

Reply:
The following assertions fail, but the code does not have an error with None

Reply:
should note their does exist a test which tests empty profile adds for int